### DISCUSSION TITLE
How to mount share in Unraid

### DISCUSSION THREAD

**Discussion by miibrito92** [slskd/slskd#982] [Q&A] [answered]
Hi, so I'm trying to configure my slsk server on unraid but I'm having a little bit of difficulty here mostly because I don't perfectly understand. Tried to mount a share folder but no luck. Changed my yaml file using this:
# directories:
#   incomplete: ~
#   downloads: ~
# shares:
#   directories:
#     - '\\192.168.1.153\media\music'
#     - ~

and also this:
# directories:
#   incomplete: ~
#   downloads: ~
# shares:
#   directories:
#     - '/music'
#     - ~

in which I added a Path like the picture in attachment.

![Captura de ecrã 2023-10-19 011413](https://github.com/slskd/slskd/assets/43927953/863188f6-fa00-47a7-993f-5b511f579355)

Any idea what I did wrong? Thanks

**Comment by Unthred** [comment, marked as answer]
I have just installed this docker and this is how I got it working on UnRaid 
Setup a variable for your music to share
![image](https://github.com/slskd/slskd/assets/5455511/63fb63aa-0565-4eeb-9d6b-1e0c6ae310ea)
Then I have a a folder mounted that points to the above variable
![image](https://github.com/slskd/slskd/assets/5455511/8396b625-6e9c-414e-bcc9-e0db63e40e79)

Then do the same kind of thing for SLSKD_DOWNLOADS_DIR and SLSKD_INCOMPLETE_DIR

Hope that makes sense

**Reply by miibrito92** [reply]
that did the trick! Thanks mate

**Reply by MagneticNeedle** [reply]
@miibrito92 mark this as the answer, that way any one looking from the main page will be able to see that this question has been answered

**Comment by PhatzDomino** [comment]
Just wanted to add my thanks for this.  Helped me massively :)

**Comment by FloydKulis** [comment]
@PhatzDomino @Unthred 
I don't quite understand. Would you show a photo from your installation?
